hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/LocalRegionFactoryDefaultTest.java

```diff
@@ -101,4 +103,40 @@ public void testEntity() {
         assertEquals(count, stats.getSecondLevelCacheMissCount());
         stats.logSummary();
     }
+
+    @Test
+    public void testEntityPropertyUpdate() {
+        final int count = 100;
+        final int childCount = 3;
+        insertDummyEntities(count, childCount);
+        sleep(1);
+        Session session = sf.openSession();
+
+        DummyEntity e1 = (DummyEntity) session.get(DummyEntity.class, (long) 1);
+        e1.setDate(new Date());
+        Set<DummyProperty> dummyProperties = e1.getProperties();
+
+
+        Transaction tx = session.beginTransaction();
+        try {
+            for (DummyProperty property : dummyProperties) {
+                property.setKey("test");
+                session.update(property);
+            }
+            tx.commit();
+        } catch (Exception e) {
+            tx.rollback();
+            e.printStackTrace();
+        } finally {
+            session.close();
+        }
+
+        session = sf2.openSession();
+        session.beginTransaction();
```

Comment:
- I see no assertion, does it mean that the failing case throws exception and if no exception then test passes ?
- in this line we begin a transaction but we do not close (commit/rollback), should we ?